mne/time_frequency/psd.py

```diff
@@ -113,21 +113,12 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
     psds : ndarray, shape (..., n_freqs)
         The power spectral densities. All dimensions up to the last will
         be the same as input unless combine is None. When combine is None
-        the PSDs are of shape (..., n_freqs, n_windows)
+        the PSDs are of shape (..., n_freqs, n_windows).
     freqs : ndarray, shape (n_freqs,)
         The frequencies.
 
     Notes
     -----
-    Please note that if using ``combine=None`` the ``psds`` output shape will
-    be different from the shape of :func:`mne.time_frequency.psd_welch`
-    ``psds`` output. This function returns windows as the last dimension of
-    ``psds`` while :func:`mne.time_frequency.psd_welch` returns windows at -3
-    dimesion. This difference is caused by the fact that possible shapes of
-    input to this function are much more variable than inputs to
-    :func:`mne.time_frequency.psd_welch` - and placing windows at -3 dimension
-    would lead to inconsistencies anyway. Enjoy!
```

Comment:
That was [covfefian enjoy](http://www.youtube.com/watch?v=C5rC8uAtiBc&t=3m0s) BTW - fortunately no longer needed thanks to @agramfort's resistance against inconsistency. :)